src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1009,6 +1009,23 @@ class TestException : Exception { }
         End Function
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
Ah, sorry. For some reason I thought it was only for internal issues. Fixed now.